src/Tizen.NUI/src/internal/Common/StringToVoidSignal.cs

```diff
@@ -89,13 +96,6 @@ public void Disconnect(System.Delegate func)
                 if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
             }
         }
-
-        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Could you  explain why Emit() is removed in more detail?

Reply:
Emit function should be used only by Bridge and there is
no reason to allow call of this function from C# api.

Moreover emit functions (as designed in c++) should take one parameter - reference to std::string (c++ reference not c# reference to string) - so current implementation of Emit is broken.

Reply:
I see :+1: